ee-feature-pack/galleon-common/src/main/resources/layers/standalone/ee-core-profile-server/layer-spec.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" ?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~
+  ~ Copyright 2023 Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="ee-core-profile-server">
+    <dependencies>
+        <layer name="core-server"/>
+        <layer name="cdi"/>
+        <layer name="jaxrs"/>
```

Comment:
Should we be explicit and add the jsonb and jsonp layers even though they're already copied in?

Reply:
I would say that having them explicit would help.

Reply:
Okay, that was my thinking as well. I'll update that.